src/QsCompiler/LanguageServer/EditorState.cs

```diff
@@ -369,10 +369,11 @@ internal Task CloseFileAsync(TextDocumentIdentifier textDocument, Action<string,
             : null;
 
         /// <summary>
-        /// Resolves additional information for the given completion item.
+        /// Resolves additional information for the given completion item. Returns the original completion item if no
+        /// additional information is available.
         /// <para/>
         /// Returns null if any parameter is null or the file given in the original completion request is invalid or
-        /// ignored, or if the completion to resolve is outdated. 
```

Comment:
So, if the information is outdated, then the original item is returned?

Reply:
By outdated, do you mean invalid (e.g., the qualified name in the completion item data does not/no longer exists)?